modules/vcf2maf/1.3/src/deblacklist_ssm.R

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env Rscript
+
+# Uses GAMBLR to remove blacklisted genes from augmented MAFs
+
+log <- file(snakemake@log[[1]], open="wt")
+sink(stdout(), type = "message")
+sink(log)
+
+library(GAMBLR)
+library(data.table)
+library(tidyverse)
+library(readr)
+library(glue)
+
+genome_build = ifelse(
+    str_detect(snakemake@wildcards[["genome_build"]], "38"), 
+    "hg38", 
+    "grch37"
+)
+
+blacklist_template = snakemake@params[["blacklist_template"]]
+project_base = snakemake@params[["project_base"]]
+seq_type_param = snakemake@params[["seq_type_blacklist"]]
+drop_threshold = snakemake@params[["threshold"]]
+
+maf = snakemake@input[["maf"]]
+
+seq_type = if (seq_type_param == "both") c("genome", "capture") else snakemake@wildcards[["seq_type"]]
+
+fread_maf(maf) %>% 
+    mutate(Chromosome = as.character(Chromosome)) %>% 
```

Comment:
I think this is how Haya had it in her version of this script in the GAMBL repo. I'm not sure if she ran into issues.

Reply:
Were you using fread_maf or perhaps this was before? If you're casting it to an integer here then the damage is already done, I would think, i.e. the non-integer rows have NA.

Reply:
I'm pretty sure I was using fread_maf. It was strange because this wasn't happening with all the samples, just a few of them. I see your point though.